### PR TITLE
Bh/spubheader break submissions tab

### PR DESCRIPTION
**Pull request by howebenhowe**
Implements #1851 

Overall CSS is fairly close to mocks. It's still missing some spacing and font styling, but I suspect much of the css will be the shared across the unfinished Instructions tab and be informed by the new `SpubheaderToolbar`, so I put it off until those other pieces are in place, when a general CSS roundup would be more timely.

### CODE REVIEW

**Comment by howebenhowe** [comment]
Per the mocks, I have used the name `SubmissionTab` for what this card refers to as the `DetailsTab`

File: client/containers/Pub/SpubHeader/SubmissionTab.tsx [hunk 1]
```diff
@@ -46,25 +48,21 @@ const TitleDescriptionAbstract = (props: Props) => {
...
-			<Label>
-				<h2>Pub Content</h2>
-				<span className="input-prompt">
-					Enter your primary submission content in the pub body below by typing or by
-					importing files in which you already have content prepared.
-				</span>
+				<h2>Default Download File</h2>
 			</Label>
-		</>
+			<p className="submission-tab-prompt-text">
```
Comment (idreyn) [inline client/containers/Pub/SpubHeader/SubmissionTab.tsx:53]: Is there a CSS declaration that uses this class?

File: client/containers/Pub/SpubHeader/SpubHeader.tsx [hunk 2]
```diff
@@ -82,29 +86,45 @@ const SpubHeader = (props: Props) => {
...
+						/>
+					</SpubHeaderTab>
+				}
+			/>
+			<Tab
+				className={`tab-panel ${maybeActiveClass('contributors')}`}
+				id="contributors"
+				title={contributorsTabTitle}
+				panel={
+					<SpubHeaderTab expandToFold>
```
Comment (idreyn) [inline client/containers/Pub/SpubHeader/SpubHeader.tsx:116]: This one shouldn't have `expandToFold`.

**Review by idreyn — APPROVED** [review]
LGTM and I'm happy to defer the CSS pass until later!

One comment: My original intention was for the `SpubHeaderTab` wrapper to be used _inside_ the `ContributorsTab`, `SubmissionTab`, etc. I'm fine with the way this is, but just throwing that out there.

**Pull request merged**